pkg/generate/app/cmd/newapp.go

```diff
@@ -526,19 +530,21 @@ func (c *AppConfig) buildPipelines(components app.ComponentReferences, environme
 
 				// Append any exposed ports from Dockerfile to input image
 				if ref.Input().Uses.IsDockerBuild() {
-					if df, err := ref.Input().Uses.Info().Dockerfile.Dockerfile(); err == nil {
-						exposed, ok := df.GetDirective("EXPOSE")
```

Comment:
Another bug here: if the image had multiple EXPOSE instructions, we ignore any but the first. Fixed in the new code below.